### ISSUE TITLE
pyreadline doesn't work with Jep on Windows

### ISSUE THREAD

**Issue by ndjensen** [mrj0/jep#23]
If a developer installs pyreadline on Windows, pyreadline works fine for the python interactive interpreter but doesn't work for the jep interactive interpreter.  I modified ctypes `__init__.py` to print out each library as it loads, and for some reason running jep can't find msvcr90.dll but running python can find msvcr90.dll.  Both have the same PATH variable.  Here is sample output:

```
C:\Users\Nate>jep
C:\Users\Nate>SETLOCAL
C:\Users\Nate>SET VIRTUAL_ENV=
C:\Users\Nate>IF NOT "" == "" (
SET PATH="\bin";"\bin";"D:\python2.7\;D:\python2.7\Scripts;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\ATI Stream\bin\x86_64;C:\Program Files (x86)\ATI Stream\bin\x
6;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files\
icrosoft SQL Server\120\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\"
 SET PYTHONHOME=
)
C:\Users\Nate>SET cp="D:\python2.7\Lib\site-packages\jep\jep-3.4.0.jar"
C:\Users\Nate>IF DEFINED CLASSPATH (SET cp="D:\python2.7\Lib\site-packages\jep\jep-3.4.0.jar"; )
C:\Users\Nate>SET jni_path="D:\python2.7\Lib\site-packages\jep"
C:\Users\Nate>SET args=
C:\Users\Nate>IF "" == "" (SET args="D:\python2.7\Lib\site-packages\jep\console.py" )
C:\Users\Nate>java -classpath "D:\python2.7\Lib\site-packages\jep\jep-3.4.0.jar" -Djava.library.path="D:\python2.7\Lib\site-packages\jep" jep.Run "D:\python2.7\Lib\site-packages\j
p\console.py"

              No readline available.

               If your platform doesn't have readline, try this:
               https://pypi.python.org/pypi/readline

>>> exit()
C:\Users\Nate>ENDLOCAL

C:\Users\Nate>python
Python 2.7.10 (default, May 23 2015, 09:44:00) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import readline
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named readline
>>> import pyreadline
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named pyreadline
>>> exit()

C:\Users\Nate>pip install pyreadline
ctype CDLL: kernel32
ctype CDLL: shell32
ctype CDLL: kernel32
ctype CDLL: None
ctype CDLL: msvcr90.dll
ctype CDLL: rpcrt4
You are using pip version 7.0.1, however version 7.1.0 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting pyreadline
Installing collected packages: pyreadline
Successfully installed pyreadline-2.0

C:\Users\Nate>jep
C:\Users\Nate>SETLOCAL
C:\Users\Nate>SET VIRTUAL_ENV=
C:\Users\Nate>IF NOT "" == "" (
SET PATH="\bin";"\bin";"D:\python2.7\;D:\python2.7\Scripts;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\ATI Stream\bin\x86_64;C:\Program Files (x86)\ATI Stream\bin\x
6;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files\
icrosoft SQL Server\120\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\"
 SET PYTHONHOME=
)
C:\Users\Nate>SET cp="D:\python2.7\Lib\site-packages\jep\jep-3.4.0.jar"
C:\Users\Nate>IF DEFINED CLASSPATH (SET cp="D:\python2.7\Lib\site-packages\jep\jep-3.4.0.jar"; )
C:\Users\Nate>SET jni_path="D:\python2.7\Lib\site-packages\jep"
C:\Users\Nate>SET args=
C:\Users\Nate>IF "" == "" (SET args="D:\python2.7\Lib\site-packages\jep\console.py" )
C:\Users\Nate>java -classpath "D:\python2.7\Lib\site-packages\jep\jep-3.4.0.jar" -Djava.library.path="D:\python2.7\Lib\site-packages\jep" jep.Run "D:\python2.7\Lib\site-packages\j
p\console.py"
ctype CDLL: kernel32
ctype CDLL: user32
ctype CDLL: msvcrt
ctype CDLL: msvcr90.dll
Windows error importing readline: [Error 126] The specified module could not be found
>>> exit()
C:\Users\Nate>ENDLOCAL

C:\Users\Nate>python
Python 2.7.10 (default, May 23 2015, 09:44:00) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
ctype CDLL: kernel32
ctype CDLL: user32
ctype CDLL: msvcrt
ctype CDLL: msvcr90.dll
ctype CDLL: python27
>>> import readline
>>> import pyreadline
>>> exit()
```

**Comment by ndjensen** [comment]
Ok I traced this down to this line: https://github.com/pyreadline/pyreadline/blob/1.7/pyreadline/console/console.py#L605 where it does `cdll.LoadLibrary()`.  Interestingly enough, the same line of code does not exist in the master branch and pyreadline seems to mostly work on the master branch.  Looking into it deeper, the pypi page for pyreadline is quite out of date and behind the master branch, which is what gets installed with pip.

I've partially corrected this here: https://github.com/mrj0/jep/commit/1a94fe7c90fe08579cffc1ae002a2d16db0ee860

Need to update docs to indicate don't use pip to get pyreadline, and test further on windows.

**Comment by ndjensen** [comment]
I've tested and it works nicely on Windows if you use the master branch from pyreadline on github instead of the version on pypi.

**Comment by ndjensen** [comment]
Fixed on 3.4.2.
